config/initializers/stripe.rb

```diff
@@ -0,0 +1 @@
+Stripe.api_key = Rails.env.production? ? ENV['STRIPE_LIVE_SECRET_KEY'] : ENV['stripe_secret_key']
```

Comment:
Figaro handles this for you... You set the variables based on the environment in your yaml file.